ChaosMod/Components/MetaModifiers.h

```diff
@@ -13,4 +14,7 @@ class MetaModifiers : public Component
 	bool HideChaosUI                         = false;
 	bool DisableChaos                        = false;
 	bool FlipChaosUI                         = false;
+	TimerMode OverrideTimerMode              = TimerMode::None;
+	int TimeToDispatchEffect                 = 0;
+	int DistanceToDispatchEffect             = 0;
```

Comment:
Need to be exposed to the Lua runtime as well

Reply:
I'm not sure how to add enums to Lua runtime, I've added TimeToDispatchEffect and DistanceToDispatchEffect

Reply:
https://github.com/gta-chaos-mod/ChaosModV/blob/53b281c5f77116b7748d1f7526925ddba71753a0/ChaosMod/Components/LuaScripts.cpp#L612

Like this.